.cookiecutter/cookiecutter.json

```diff
@@ -8,6 +8,12 @@
     "report/__main__.py",
     "report/app.py",
     "report/core.py",
+    "report/db.py",
+    "report/models/__init__.py",
+    "tests/pytest_plugins/db.py",
+    "tests/pytest_plugins/factory_boy.py",
+    "tests/unit/pytest_plugins/db.py",
+    "tests/functional/pytest_plugins/db.py",
```

Comment:
These files from the cookiecutter don't look like they apply/are needed in report. We can always remove them from this list if we decide they're useful